README.md

```diff
@@ -67,8 +67,9 @@ export const configs = {
 };
 ```
 
-4. Start the bot
-   `deno run --allow-read --allow-write --allow-net --import-map=./import-map.json mod.ts`
+4. Start the bot using either:
+* `deno run -A --quiet mod.ts` or
+* `deno run -A --quiet --unstable --watch mod.ts` to enable automatic reloading on changes
```

Comment:
Is this needed? Most of the time you can use eg. `!reload commands` to load your changes. Restarting the bot on every file change can be time consuming and is not recommended (but most likely you won't hit the 1000 identify limit).

Reply:
`!reload` command does not really automate the command, you are still "running" the command to reload all of the commands; on the other hand, `--watch` flag restarts the process once a file is updated and saved...